packages/neos-ui-guest-frame/src/InlineUI/NodeToolbar/index.js

```diff
@@ -37,10 +37,7 @@ export default class NodeToolbar extends PureComponent {
         isSticky: false
     };
 
-    constructor() {
-        super();
-        this.iframeWindow = document.getElementsByName('neos-content-main')[0].contentWindow;
-    }
+    iframeWindow = document.getElementsByName('neos-content-main')[0].contentWindow;
```

Comment:
Maybe we can use this opportunity to replace this call with the helper method from `@neos-project/neos-ui-guest-frame/src/dom`